### PR TITLE
SLO & MS Acc Tests Fixes

### PR DESCRIPTION
**Pull request by sathish-soundarapandian**
## Describe your changes
1. Intermittent test case failures - Added timeout to fix it.
2. Fix SLO test due to recent change wherein name of the service in the YAML should match with the one in request.
3. Add DeleteUnderlyingResource test for SLO & MS resource.

## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- Sub Category Field Check: `trigger subcategoryfieldcheck`

### CODE REVIEW

File: internal/service/platform/monitored_service/resource_monitored_service.go [hunk 1]
```diff
@@ -258,6 +258,11 @@ func resourceMonitoredServiceRead(ctx context.Context, d *schema.ResourceData, m
 	resp, httpResp, err := c.MonitoredServiceApi.GetMonitoredService(ctx, identifier, accountIdentifier, orgIdentifier, projectIdentifier)
 
 	if err != nil {
+		if err.Error() == "400 Bad Request" {
```
Comment (rathodmeetsatish) [inline internal/service/platform/monitored_service/resource_monitored_service.go:261]: 400 Bad Request is too general (This condition is only to be applied when we are not able to find entity) does the api return RESOURCE_NOT_FOUND error code or 404 status?

Comment (sathish-soundarapandian) [reply internal/service/platform/monitored_service/resource_monitored_service.go:261]: Yeah, it's too general. But that's how the API is written (kind of legacy).
```
MonitoredService monitoredServiceEntity = getMonitoredService(projectParams, identifier);
if (monitoredServiceEntity == null) {
  throw new InvalidRequestException(
      String.format("Monitored Source Entity with identifier %s is not present", identifier));
}
```
Mostly, team will be moving this to follow proper convention soon. In the meantime, I have added this (Specifically for __DeleteUnderlyingResource_ test)
Let me know whether to keep the changes or else we can comment it out for now and then enable as and when the source API is fixed

Comment (rathodmeetsatish) [reply internal/service/platform/monitored_service/resource_monitored_service.go:261]: yes please comment the changes and uncomment when the api returns something specfic (error code/status code) for entity not found

Comment (sathish-soundarapandian) [reply internal/service/platform/monitored_service/resource_monitored_service.go]: Done

**Review by rathodmeetsatish — APPROVED** [review]

**Pull request merged**